lms/djangoapps/django_comment_client/management/tests.py

```diff
@@ -0,0 +1,245 @@
+"""
+UnitTests for the rename_user function in the management_utils module
+"""
+from django.contrib.auth.models import User
+from pymongo import MongoClient
+from student.tests.factories import UserFactory
+import datetime
+from django_comment_client.management import management_utils
+from django.test import TestCase
+
+
+class TestRenameUser(TestCase):
+    """
+    These tests can be run from the terminal using the following command:
+    python ./manage.py lms test --verbosity=1 lms/djangoapps/django_comment_client/management/tests.py  --traceback --settings=test
+    """
+    NEW_USERNAME = 'targetName'
+
+    @classmethod
+    def setUpClass(cls):
+        client = MongoClient()
+        # chooses to write to the forum database
+        cls.db = client.forum
+        # maintain arrays of mongo data so that we can delete them when the tests are done
+        cls.mongo_user_ids = []
+        cls.mongo_comment_ids = []
+        cls.sql_users = []
+        cls.seed_sql_users()
+        cls.seed_mongo_comments()
+        cls.seed_mongo_users()
+
+    @classmethod
+    def seed_sql_users(cls):
+        """
+        creates users in the test sql database
+        """
+        cls.sql_users.append(UserFactory.create())
+        cls.sql_users.append(UserFactory.create())
+
+    def setUp(self):
+        """
+        re-queries the sql_users between each test
+        """
+        self.sql_users[0] = User.objects.get(id=self.sql_users[0].id)
+        self.sql_users[1] = User.objects.get(id=self.sql_users[1].id)
+
+    @classmethod
+    def seed_mongo_comments(cls):
+        """
+        creates test comments in the contents collection of the mongo database
+        """
+        comment_1 = \
```

Comment:
The `\` continuation character is superfluous. Delete the `\`, move the opening `{` to where the `\` was, then unindent the body of the dict, like:
```python
comment = {
    "_type": "CommentThread",
}
```